Blackjack Betting Game

```diff
@@ -109,18 +110,37 @@ def play_game():
         
         # Play again player prompt
         play_cont = 1
+        if player_chips <= 0:
+             print("Game over! You're out of chips.")
         while True:
             choice = input('Do you want to play again? [Y/N]')
             if choice.lower() == 'y':
                 play_cont = 1
                 break
             elif choice.lower() == 'n':
-                print('Thanks for playing!')
```

Comment:
parang di na natin need neto kase meron na sa line 138

Reply:
Paltan mo nalang ng "Returning to Main Menu, Press Enter to Continue..."